.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 examples/remount/assets
+.idea/
```

Comment:
@jedireza Webstorm/IntelliJ - One of the side effects of it is this folder is added to projects - as I was making quite a few commits I didn't want to accidental add it as it's just workspace specific settings and things like code completion.

See https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore for how noisy it can be.